as-modules/embedded/src/main/resources/org/jgroups/kubernetes/main/module.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<module xmlns="urn:jboss:module:1.3" name="org.jgroups.kubernetes.kubernetes">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${org.jgroups.kubernetes:kubernetes}"/>
+        <artifact name="${org.jgroups.kubernetes:common}"/>
```

Comment:
Interesting piece for review: All `KUBE_PING` bits are in 1 module. There are [plans for refactoring this protocol](https://issues.jboss.org/browse/CLOUD-662), so I would like to limit the scope of changes on our side